### PR TITLE
Notify support documentation

### PR DESCRIPTION
**Pull request by sarahseewhy**
### WHAT
Documents a base set of tasks for supporting Notify:

* Accessing metrics
* Accessing logs
* Testing email/sms
* Rolling AMIs
* Running Ansible

The documentation is an expanded form of the notes I took at the [Notify Game Day session](https://hackmd.cloudapps.digital/SkCAz-KFT3qGFT5QaSybkw).

### CODE REVIEW

File: source/documentation/notify.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+- [Document Download](https://grafana-paas.cloudapps.digital/d/FwXIHjiiz)
+
+- [Notify Apps](https://grafana-paas.cloudapps.digital/d/aCYK0WDik)
+
+- [Postgres](https://grafana-paas.cloudapps.digital/d/SYlv1gAmz)
+
+
+### CloudWatch dashboards
+
+**Prerequisites: the [GDS CLI](https://github.com/alphagov/gds-cli) and [YubiKey](https://re-team-manual.cloudapps.digital/yubikeys.html#yubikeys) setup are required to access CloudWatch metrics.**
```
Comment (paroxp) [inline source/documentation/notify.md]: It's not strictly required, but would make people lives easier. 🤔

File: source/documentation/notify.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+**Prerequisites: the [GDS CLI](https://github.com/alphagov/gds-cli) and [YubiKey](https://re-team-manual.cloudapps.digital/yubikeys.html#yubikeys) setup are required to access CloudWatch metrics.** 
+
+Once GDS CLI and YubiKey configuration is set up, open a new terminal window and run:
+
+```
+$ gds aws <some-notify-service>
+```
+
+This command will prompt a GPG key sign-in process.
```
Comment (paroxp) [inline source/documentation/notify.md]:
```suggestion
This command will prompt a sign-in process with the use of MFA stored on a YubiKey.
```

File: source/documentation/notify.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+Follow the RE Team Manual's ["Getting started with Logit"](https://reliability-engineering.cloudapps.digital/logging.html#content) instructions to access Kibana logs.
+
+### CloudWatch
+
+Follow the accessing [CloudWatch dashboards](#cloudwatch-dashboards) instructions.
+
+## Testing SMS and Email
+
+**Prerequisites: a Notify Admin account is required to test SMS and email notifications.**
```
Comment (paroxp) [inline source/documentation/notify.md]:
```suggestion
**Prerequisites: an account on Notifications admin portal with access to any organisation is required to test SMS and email notifications.**
```

File: source/documentation/notify.md (hunk 1 above)
Comment (sarahseewhy) [reply source/documentation/notify.md]: I updated the branch with a commit to reflect these suggestions. Marking this as resolved.

**Pull request merged**